app/views/RoomsListView/index.tsx

```diff
@@ -359,9 +362,9 @@ class RoomsListView extends React.Component<IRoomsListViewProps, IRoomsListViewS
 		) {
 			this.getSubscriptions();
 		}
-		// Update current item in case of another action triggers an update on rooms reducer
-		if (isMasterDetail && rooms[0] && item?.rid !== rooms[0] && !dequal(rooms, prevProps.rooms)) {
-			this.setState({ item: { rid: rooms[0] } as ISubscription });
+		// Update current item in case of another action triggers an update on room subscribed reducer
+		if (isMasterDetail && roomSubscribed && item?.rid !== roomSubscribed && roomSubscribed !== prevProps.roomSubscribed) {
+			this.setState({ item: { rid: roomSubscribed } as ISubscription });
```

Comment:
If we have this value on the Redux, why do we need it on the local state?

Reply:
We use this `ITEM` inside the function `goOtherRoom`, but I was unable to test it correctly, so I think we can modify this, when we fix the iPAD key commands